_layouts/default.html

```diff
@@ -12,8 +12,7 @@
   {% capture prefer_mode %}data-mode="{{ site.theme_mode }}"{% endcapture %}
 {% endif %}
 
-<!-- `site.alt_lang` can specify a language different from the UI -->
-<html lang="{{ site.alt_lang | default: site.lang }}" {{ prefer_mode }}>
+<html lang="{{ lang }}" {{ prefer_mode }}>
```

Comment:
In a way `page.lang` and `site.alt_lang` serve a similar purpose, but `page.lang` should have a higher priority than `site.alt_lang`, so I now think it should be changed to something like the following:

```suggestion
<!-- `site.alt_lang` can specify a language different from the UI -->
	<html lang="{{ page.lang | default: site.alt_lang | default: site.lang }}" {{ prefer_mode }}>
```